webservices/tasks/download.py

```diff
@@ -192,6 +192,6 @@ def export_query(path, qs):
 @app.task
 def clear_bucket():
     for obj in task_utils.get_bucket().objects.all():
-        if not obj.key.startswith('legal') or not obj.key.startswith('bulk-downloads'):
+        if not obj.key.startswith('legal') and not obj.key.startswith('bulk-downloads'):
```

Comment:
Note that the `set` of prefixes to check for should be defined before the `for` loop, and please feel free to use a better/more descriptive name than `matches`!